src/genomicsdb_processor.h

```diff
@@ -92,15 +92,26 @@ class ColumnarVariantCallProcessor : public GenomicsDBVariantCallProcessor {
                const int64_t* coordinates,
                const genomic_interval_t& genomic_interval,
                const std::vector<genomic_field_t>& genomic_fields);
+  void process_str_field(const std::string& field_name, PyObject *calls, int dims, npy_intp *sizes) {
+    auto found = std::find(m_field_names.begin(), m_field_names.end(), field_name);
+    if (found != m_field_names.end()) {
+        PyDict_SetItem(calls, PyUnicode_FromString(field_name.c_str()),
+                       PyArray_SimpleNewFromData(dims, sizes, NPY_OBJECT, m_string_fields[field_name].data()));
+    }
+  }
   PyObject* construct_data_frame() {
     int dims = 1;
     npy_intp sizes[1] = { static_cast<npy_intp>(m_sample_names.size()) };
     PyObject *calls = PyDict_New();
     PyDict_SetItem(calls, PyUnicode_FromString("Sample"), PyArray_SimpleNewFromData(dims, sizes, NPY_OBJECT, m_sample_names.data()));
     PyDict_SetItem(calls, PyUnicode_FromString("CHR"), PyArray_SimpleNewFromData(dims, sizes, NPY_OBJECT, m_chrom.data()));
     PyDict_SetItem(calls, PyUnicode_FromString("POS"), PyArray_SimpleNewFromData(dims, sizes, NPY_INT64, m_pos.data()));
-
+    // Process REF, ALT and GT first.
+    process_str_field("REF", calls, dims, sizes);
+    process_str_field("ALT", calls, dims, sizes);
+    process_str_field("GT", calls, dims, sizes);
```

Comment:
Naive question: can you tell me why we process REF, ALT, GT first instead of in the loop?

Reply:
They could be in any order based on how GenomicsDB processes them. We could let the loop handle it, but sometimes, the order is `ALT,REF,..,GT`. I just thought it nicer to have `REF` before `ALT`, guess it does not matter for `GT` as much.. But, I can process them in the loop if that is OK too. Let me know.

Reply:
Yeah, can't decide. My initial concern was that this bit of code gets invoked for each variant so trying to minimize any extraneous code. But that's probably micro-optimization...thoughts?

Reply:
Ah - thanks for that clarification, I misread this as being part of `process` somehow. I think this is good to merge the way it is if you want.